qgepplugin/swmm/swmm_full_report.qgs

```diff
@@ -0,0 +1,8542 @@
+<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
```

Comment:
What is this file for, I don't see a reference elsewhere in the code.

Reply:
Documentation is here: https://qgep.github.io/docs/qgep_swmm/Extract-Results.html